server/push/router/push.router.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Application from '../../base/application.js';
+import {sendPush} from '../controller/send_push';
+
+@Application.route('/api/push/')
+export default class PushRouter {
+  /**
+   * Send push with payload to endpoint with it's auth / ecdh key
+   * @param {json} request request object
+                   - request.body.endpoint endpoint endpoint from subscription
+                   - request.body.keys keys authentication key
+                     and p256dh key from subscription
+                   - request.body.payload payload payload data send to endpoint
+   * @param {json} response response object
+   */
+  post(request, response) {
+    if (request.body.endpoint && request.body.keys) {
+      sendPush(request.body.endpoint, request.body.keys,
+        JSON.stringify(request.body.payload))
+        .then((result) => {
+          response.send(result);
+        }).catch((error) => {
+          response.send(error);
+        });
+    } else {
+      response.sendStatus(400);
```

Comment:
IMHO, From newly updated RFC, 400 Bad Request can apply to here
>    The 400 (Bad Request) status code indicates that the server cannot or
   will not process the request due to something that is perceived to be
   a client error (e.g., malformed request syntax, invalid request
   message framing, or deceptive request routing).

on the other hands,
> The 403 (Forbidden) status code indicates that the server understood
   the request but refuses to authorize it.  A server that wishes to
   make public why the request has been forbidden can describe that
   reason in the response payload (if any).
 https://tools.ietf.org/html/rfc7231#page-58

So in this case we can apply malformed request syntax here